WalletWasabi.Packager/Program.cs

```diff
@@ -367,10 +384,34 @@ private static async Task PublishAsync()
 					continue;
 				}
 
-				ZipFile.CreateFromDirectory(currentBinDistDirectory, Path.Combine(BinDistDirectory, $"Wasabi-osx-{VersionPrefix}.zip"));
+				var zipFileName = target.Contains("arm64")
+					? $"_WasabiToMac-osxarm64-{VersionPrefix}.zip"
+					: $"_WasabiToMac-osx-{VersionPrefix}.zip";
+
+				var zipFilePath = Path.Combine(BinDistDirectory, zipFileName);
+
+				ZipFile.CreateFromDirectory(currentBinDistDirectory, zipFilePath);
 
 				await IoHelpers.TryDeleteDirectoryAsync(currentBinDistDirectory).ConfigureAwait(false);
 				Console.WriteLine($"Deleted {currentBinDistDirectory}");
+
+				try
+				{
+					Console.WriteLine($"Trying to copy unsigned zip file to removable.");
+					var driveList = DriveInfo.GetDrives().Where(d => d.DriveType == DriveType.Removable);
+					if (!driveList.Any())
+					{
+						Console.WriteLine($"No removable drive found - finish.");
+					}
+
+					var drive = driveList.Single();
+					var targetFilePath = Path.Combine(drive.Name, zipFileName);
+					File.Move(zipFilePath, targetFilePath, overwrite: true);
+				}
+				catch (Exception ex)
+				{
+					Console.WriteLine($"There was an error during copying the file to removable: '{ex.Message}'");
```

Comment:
Would it be better to have stacktrace here:

```suggestion
					Console.WriteLine($"There was an error during copying the file to removable: '{ex}'");
```

?

Reply:
It is enough here.